packages/shared/components/atoms/labels/AccountLabel.svelte

```diff
@@ -8,7 +9,7 @@
 
 <div class="flex flex-row items-center space-x-3">
     <ColoredCircle color={account?.color} />
-    <Text fontSize="14" fontWeight={FontWeight.semibold} classes={selected ? '' : 'opacity-50'}>
+    <Text fontSize={$mobile ? '16' : '14'} fontWeight={FontWeight.semibold} classes={selected ? '' : 'opacity-50'}>
```

Comment:
we should try avoiding using `$mobile ? this : that` from now on, the new coding conventions suggest adding variations to the shared components instead of using the platform conditionals 🙏🏼

Reply:
thanks! have just fixed it